src/CMakeLists.txt

```diff
@@ -41,7 +41,8 @@ setup_target(libvgmstream)
 
 # Set up the proper include directories
 target_include_directories(libvgmstream PRIVATE
-	${CMAKE_CURRENT_SOURCE_DIR}
+	${VGM_SOURCE_DIR}
```

Comment:
```suggestion
	${VGM_BINARY_DIR}
```

Reply:
Doesn't seem that this lib requires such include.
I can build it without this path, please confirm.

Reply:
If it build fine without it, then it is fine to leave it out. I could've sworn there was a reason I had it set... (and it should've been SOURCE, not BINARY, my suggested change is wrong in this case).

Reply:
It's already removed.
I think this MR can be resolved.